### PR TITLE
Set layerStyle defaultvalue on create

### PR DESCRIPTION
**Pull request by dtrucs**

### CODE REVIEW

File: src/modules/RA/DataLayer/components/DataLayerTabbedForm.js [hunk 1]
```diff
@@ -50,10 +52,21 @@ const CustomFormTab = ({ hidden, ...props }) => (
     : <FormTab hidden={hidden} {...props} />
 );
 
-const DataLayerTabbedForm = ({ classes, translate, ...props }) => (
+const DataLayerTabbedForm = ({
+  classes,
+  translate,
+  randomColor,
+  sourceData: { geom_type: type = POLYGON } = {},
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerTabbedForm.js]:
```suggestion
  sourceData: { geom_type: geomType = POLYGON } = {},
```

File: src/modules/RA/DataLayer/components/DataLayerTabbedForm.js [hunk 2]
```diff
@@ -50,10 +52,21 @@ const CustomFormTab = ({ hidden, ...props }) => (
...
+  ...props
+}) => (
   <>
     <SourceFetcher />
-    <TabbedForm {...props}>
+    <TabbedForm
+      defaultValue={{
+        layer_style: getLayerStyleDefaultValue(randomColor, type),
```
Comment (mabhub) [inline src/modules/RA/DataLayer/components/DataLayerTabbedForm.js]:
```suggestion
        layer_style: getLayerStyleDefaultValue(randomColor, geomType),
```

File: src/utils/geom.js [hunk 3]
```diff
@@ -7,6 +7,29 @@ export const MULTI_LINESTRING = 5;
...
+export const getLayerStyleDefaultValue = (color, geomType) => {
+  const type = getShapeFromGeomType(geomType);
+  return {
+    type,
+    paint: {
+      [`${type}-color`]: color,
+    },
+  };
+};
+
```
Comment (mabhub) [inline src/utils/geom.js]: Just a suggestion, not mandatory:

```jsx
export const getLayerStyleDefaultValue_ = (
  color,
  geomType,
  type = getShapeFromGeomType(geomType),
) => ({
  type,
  paint: {
    [`${type}-color`]: color,
  },
});
```

Comment (dtrucs) [reply src/utils/geom.js]: I'm not sure about this one.
The `geomType` argument is only useful for getting the shape, we don't need it for anything else.
If we set the third parameters `type`, the `geomType` in the middle seems confusing.

Comment (mabhub) [reply src/utils/geom.js]: Could also be done with object as function arg:

```jsx
export const getLayerStyleDefaultValue = ({
  color,
  geomType,
  type = getShapeFromGeomType(geomType),
}) => ({
  type,
  paint: {
    [`${type}-color`]: color,
  },
});
```

**Pull request merged**